bot/utils/checks.py

```diff
@@ -67,24 +70,43 @@ def in_whitelist_check(
         # categories, it's probably not wise to rely on its category in any case.
         channels = tuple(channels) + (redirect,)
 
+    if every:
+        log.trace(
+            f"{ctx.author} may use the `{ctx.command.name}` command "
+            f"as it is completely whitelisted by every."
```

Comment:
```suggestion
            f"as it is completely whitelisted by 'every'."
```
We want to make it clear that we're referring to the `every` keyword argument.